serializers/ideas.js

```diff
@@ -7,7 +7,8 @@ const config = require(path.resolve('./config'));
 
 const ideaSerializer = new Serializer('ideas', {
   id: 'id',
-  attributes: ['title', 'detail', 'created', 'creator', 'ideaTags'],
+  // TODO where to put rate in this serialization
+  attributes: ['title', 'detail', 'created', 'creator', 'ideaTags', 'rate'],
```

Comment:
I have an impression that `rate` or whatever we want sum of votes to be called, should belong to `meta`. See the related issue #59.

It can be implemented with `jsonapi-serializer` something like this:

```
{
  dataMeta {
    voteSum(record, current) {
      return current.rate;
    }
  }
}
```

The examples are here: https://github.com/SeyZ/jsonapi-serializer/blob/master/test/serializer.js#L254
Are you opinionated on this, @agatatalita ?

Reply:
No, I actually wanted to include it to meta.